src/bin/submitpackage/main.rs

```diff
@@ -0,0 +1,78 @@
+use base64::{engine::general_purpose, Engine as _};
+use bitcoin::network::Magic;
+use clap::{Parser, ValueEnum};
+use hex_string::HexString;
+use nostr::prelude::*;
+use nostr::Keys;
+use nostr_sdk::Client;
+
+#[derive(Parser)]
+#[command()]
+struct Args {
+    #[arg(short, value_enum)]
+    network: Network,
```

Comment:
Can just use `rust-bitcoin`'s enum

```suggestion
    #[clap(default_value_t = Network::Bitcoin, short, long)]
    /// Network the transaction are for ["bitcoin", "testnet", "signet, "regtest"]
    pub network: Network,
```

Reply:
Done. Downside seems to be that the options aren't shown in the cli help anymore.

```
Options:
  -n, --network <NETWORK>  [default: bitcoin]
```